container_files/assets/www/doc/builtin/functions/EM.md

```diff
@@ -0,0 +1,40 @@
+# EM Function
```

Comment:
I wouldn't call this EM; that's too general a term. I'd call it gaussian clustering.  The fact that it's trained by the EM algorithm is just an implementation detail; it would be like calling it "C++ not kmeans clustering".  True, but not really important.